### PR TITLE
Allow moving of variants between genes in variant revise form

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: client/src/app/components/revisions/revision-list/revision-list.component.html [hunk 1]
```diff
@@ -11,6 +11,16 @@
         [nzActive]="true"
       >
         <ng-container [ngSwitch]="revision.fieldName">
+          <ng-container *ngSwitchCase="'gene_id'">
+            <cvc-revision-list-diff
+              [diffObject]="revision.linkoutData.diffValue"
+              entityType="Gene">
+              {{revision.linkoutData.diffValue | json}}
```
Comment (acoffman) [inline client/src/app/components/revisions/revision-list/revision-list.component.html:18]: I'm guessing we want to remove this/it was for debugging?

**Review by acoffman — APPROVED** [review]
LGTM aside from the one comment

File: client/src/app/components/revisions/revision-list/revision-list.component.html (hunk 1 above)
Comment (susannasiebert) [inline client/src/app/components/revisions/revision-list/revision-list.component.html:18]:
```suggestion
```

Comment (susannasiebert) [reply client/src/app/components/revisions/revision-list/revision-list.component.html]: Good catch. I pushed a commit to remove this line.

**Pull request merged**